packages/frontend/core/src/modules/collection-rules/services/collection-rules.ts

```
@@ -75,17 +98,48 @@ export class CollectionRulesService extends Service {
-                new Set<string>(extraAllowList ?? [])
-              );
-
               return {
-                filtered: finalSet,
+                filtered: filtered,
                 filterErrors: results.map(i => ('error' in i ? i.error : null)),
               };
             })
           );
 
+    const extraFiltered$ =
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)